lang/java/avro/src/main/java/org/apache/avro/io/JsonDecoder.java

```diff
@@ -90,12 +89,11 @@ private static Symbol getSymbol(Schema schema) {
    * 
    * @param in The InputStream to read from. Cannot be null.
    * @throws IOException
+   * @throws NullPointerException if {@code in} is {@code null}
    * @return this JsonDecoder
    */
   public JsonDecoder configure(InputStream in) throws IOException {
-    if (null == in) {
-      throw new NullPointerException("InputStream to read from cannot be null!");
-    }
+    Objects.requireNonNull(in, "InputStream to read from cannot be null");
```

Comment:
```suggestion
    Objects.requireNonNull(in, "InputStream cannot be null");
```

Reply:
@RyanSkraba Thank for your interest and review!!!

Requested change made!!!